src/createUiDefinition.json

```diff
@@ -226,6 +226,208 @@
                 ]
             },
 
+            {
+                "name": "customDNSStep",
+                "label": "SSL Certificate",
+                "bladeTitle": "Configure SSL Renewal",
+                "subLabel": {
+                    "preValidation": "Configure SSL renewal",
+                    "postValidation": "SSL renewal configuration complete"
+                },
+                "elements": [
+                    {
+                        "name": "information",
+                        "type": "Microsoft.Common.InfoBox",
+                        "options": {
+                            "icon": "Info",
+                            "text": "Set the time at which SSL certificate renewal will take place. Please note that the times are specified in UTC."
```

Comment:
I'm wondering if we should expose this, as its a technical detail the user wont (or shouldn't) care about. We could set it to a fixed time (like 00:30) and remove that time/window from the available backup slots?

Reply:
I have exposed this because if the certificate needs renewing it will restart the service to handle the renewal. As this could impact the user's service and there are time differences to consider I wanted to provide the ability for it to be changed to a time that will have less impact.